feature-libs/product-configurator/rulebased/root/cpq/cpq-configurator-layout.module.ts

```diff
@@ -16,10 +16,10 @@ import { CpqConfiguratorPageLayoutHandler } from './cpq-configurator-page-layout
         CpqConfigurationTemplate: {
           header: {
             md: {
-              slots: ['SiteLogo', 'MiniCart'],
+              slots: ['SiteLogo', 'VariantConfigExitButton', 'MiniCart'],
```

Comment:
Slot should be named CpqConfigExitButton

Reply:
Done